Documentation/Configuration/Typo3ConfVars/SYS.rst

```diff
@@ -1097,3 +1097,37 @@ routing
 
     ..  seealso::
         :ref:`message-bus-routing`
+
+FileInfo
+========
+
+.. index::
+   TYPO3_CONF_VARS SYS; FileInfo fileExtensionToMimeType
```

Comment:
I don't understand the `:` in between

Reply:
This is the general syntax: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-index

Reply:
This is the same as the other on that page. Is there a specific question?